app/helpers/application_helper/button/cockpit_console.rb

```diff
@@ -4,6 +4,7 @@ class ApplicationHelper::Button::CockpitConsole < ApplicationHelper::Button::Bas
   def disabled?
     record_type = @record.respond_to?(:current_state) ? _('VM') : _('Container Node')
     @error_message = _("The web-based console is not available because the %{record_type} is not powered on" % {:record_type => record_type}) unless on?
+    @error_message = _('The web-based console is not available because the Windows platform is not supported') unless platform_supported? && @record.respond_to?(:current_state)
```

Comment:
`:current_state` is already being looked at in `platform_supported?` right? There's no need to test it twice.